internal/cli/test.go

```diff
@@ -99,56 +92,29 @@ func testLoginCmd(cli *cli) *cobra.Command {
 		Long:  "Launch a browser to try out your Universal Login box.",
 		Example: `  auth0 test login
   auth0 test login <client-id>
-  auth0 test login <client-id> --connection <connection>
-  auth0 test login <client-id> --connection <connection> --audience <audience>
-  auth0 test login <client-id> --connection <connection> --audience <audience> --domain <domain>
-  auth0 test login <client-id> --connection <connection> --audience <audience> --domain <domain> --scopes <scope1,scope2>
-  auth0 test login <client-id> -c <connection> -a <audience> -d <domain> -s <scope1,scope2> --force
-  auth0 test login <client-id> -c <connection> -a <audience> -d <domain> -s <scope1,scope2> --json
-  auth0 test login <client-id> -c <connection> -a <audience> -d <domain> -s <scope1,scope2> --force --json`,
+  auth0 test login <client-id> --connection-name <connection-name>
+  auth0 test login <client-id> --connection-name <connection-name> --audience <api-identifier>
+  auth0 test login <client-id> --connection-name <connection-name> --audience <api-identifier> --domain <domain>
+  auth0 test login <client-id> --connection-name <connection-name> --audience <api-identifier> --domain <domain> --scopes <scope1,scope2>
+  auth0 test login <client-id> -c <connection-name> -a <api-identifier> -d <domain> -s <scope1,scope2> --force
```

Comment:
`<api-identifier>` is `<api-audience|api-identifier>` in `auth0 test token`. Should we have similar examples here?

Reply:
Updated in [`03fac70` (#629)](https://github.com/auth0/auth0-cli/pull/629/commits/03fac70a8be5393dcaa4f7bbcb0643156266d357)